numpy/core/src/umath/loops_unary.dispatch.c.src

```diff
@@ -0,0 +1,367 @@
+/*@targets
```

Comment:
Looks like this file needs `#define NPY_NO_DEPRECATED_API NPY_API_VERSION` at the top to avoid the deprecation warning (which gets turned into an error in CI).

Reply:
It probably also needs these defines that are present in the other `loops_*.c.src` files:
```
#define _UMATHMODULE
#define _MULTIARRAYMODULE
```